lua/nvim-lightbulb.lua

```diff
@@ -92,8 +92,8 @@ local function _update_virtual_text(text, line, bufnr)
     vim.api.nvim_buf_clear_namespace(bufnr, LIGHTBULB_VIRTUAL_TEXT_NS, 0, -1)
 
     if line then
-        vim.api.nvim_buf_set_virtual_text(
-            bufnr, LIGHTBULB_VIRTUAL_TEXT_NS, line, {{text, LIGHTBULB_VIRTUAL_TEXT_HL}}, {}
+        vim.api.nvim_buf_set_extmark(
+            bufnr, LIGHTBULB_VIRTUAL_TEXT_NS, line, -1, { virt_text = {{ text }}, hl_group = LIGHTBULB_VIRTUAL_TEXT_HL, hl_mode = 'combine' }
```

Comment:
`hl_mode` "currently only affects virt_text highlights, but might affect hl_group in later versions."
As of the latest neovim master, this is still the case, so the highlight group needs to be passed together with `text` in `virt_text`.

Reply:
good point, I changed it back